src/Microsoft.Sbom.Api/Utils/Constants.cs

```diff
@@ -71,6 +71,7 @@ public static class Constants
     public const string CatalogFileName = "manifest.cat";
     public const string BsiFileName = "bsi.json";
     public const string DeleteManifestDirBoolVariableName = "DeleteManifestDirIfPresent";
+    public const string RootPackageIdValue = "SPDXRef-RootPackage";
```

Comment:
This constant is defined in both SPDX2 and SPDX3. That suggests that we're handling things inconsistently. I'm not entirely sure of the best fix. Making the parsers depend on another package doesn't seem great, but neither does having 3 copies of the same data.

Reply:
Moved to Api constants